scripts/dumps/bulk_genome_dump_from_core.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env python
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Wrapper of dump_genome_from_core.pl to dump a list of FASTA files
+"""
+import argparse
+import subprocess
+import os
+import sys
+import logging
+import shlex
+import yaml
+
+def setup_logging():
+    """
+    Sets up logging configuration.
+    """
+    logging.basicConfig(
+        level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+    )
+
+
+def detect_job_scheduler():
+    """
+    Detect if the system is using SLURM, LSF, or no job scheduler.
+
+    Returns:
+        str: The name of the detected job scheduler ('SLURM', 'LSF') or 'NONE' if no known scheduler is found.
+    """
+
+    schedulers = {
+        "SLURM": "srun",
+        "LSF": "bsub",
+    }
+
+    for scheduler, command in schedulers.items():
+        if subprocess.run(["which", command], check=False).returncode == 0:
+            return scheduler
+
+    logging.error("No job scheduler detected.")
+    sys.exit(1)
+
+
+def subprocess_call(
+    command,
+    stdout_file="/dev/null",
+    stderr_file="/dev/null",
+    work_dir=None,
+    shell=False,
```

Comment:
These parameters are not currently being used, so I'd suggest to remove them until they're needed.
```suggestion
```